Loop/Settings/SettingsView.swift

```diff
@@ -47,8 +48,18 @@ struct SettingsView: View {
                 .frame(width: 500)
                 .frame(minHeight: 500, maxHeight: 680)
 
-            MoreSettingsView()
+            ExcludeListSettingsView()
                 .tag(5)
+                .tabItem {
+                    Image(systemName: "xmark.rectangle")
+                    Text("Black list")
```

Comment:
Replace "Black list" with something like "Excluded Apps" so that it matches the code :)